Sources/Extensions/UIView+Shadow.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  UIView+Shadow.swift
+//  MessageStackView
+//
+//  Created by Ben Shutt on 26/11/2019.
+//  Copyright © 2019 3 SIDED CUBE APP PRODUCTIONS LTD. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - UIView + Shadow
+
+extension UIView {
+    
+    /// Add shadow below a `UIView`
+    /// - Parameter setMask: Update `layer.masksToBounds` and `clipsToBounds`
+    func addShadowBelow(setMask: Bool = true) {
+        if setMask {
+            layer.masksToBounds = false
```

Comment:
AFAIK `clipsToBounds` == `layer.masksToBounds` :) The answer is from a long time ago, but I doubt they've changed it much since! https://stackoverflow.com/questions/1177775/how-is-the-relation-between-uiviews-clipstobounds-and-calayers-maskstobounds

Reply:
@simonmitchell Yeah I've personally never really known the **on screen** difference between the 2. Here setting both properties does no harm though, useful if you're reading the property I think.

Reply:
Sure, I'm cool with that :D